README.md

```diff
@@ -47,9 +48,13 @@ A curated list of tools that will help you when working with or building on-top
 - [HBase Java](https://github.com/GoogleCloudPlatform/cloud-bigtable-client) - Official Java libraries and HBase client extensions for accessing Google Cloud Bigtable.
 - [Go](https://github.com/googleapis/google-cloud-go/tree/master/bigtable) - Official implementation of the Google Cloud Bigtable Go client.
 - [PHP](https://github.com/googleapis/google-cloud-php-bigtable) - Official implementation of the Google Cloud Bigtable PHP client.
+
+### Other Client-Libraries
```

Comment:
There's also the Python HappyBase: https://happybase.readthedocs.io/en/latest/ for which there's an adapter to make it work with Bigtable: https://google-cloud-python-happybase.readthedocs.io/en/latest/